fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoanApiConstants.java

```diff
@@ -22,6 +22,8 @@
 
     String emiAmountParameterName = "fixedEmiAmount";
     String maxOutstandingBalanceParameterName = "maxOutstandingLoanBalance";
+    String revolvingPeriodStartParameterName = "revolvingPeriodStartDate";
+    String revolvingPeriodEndParameterName = "revolvingPeriodEndDate";
     String disbursementDataParameterName = "disbursementData";
     String disbursementDateParameterName = "expectedDisbursementDate";
```

Comment:
do you expose all the new fields for the client?

Reply:
I don't know. Getting into it now...